common/thorhelper/thorsoapcall.cpp

```diff
@@ -632,7 +632,7 @@ class CMatchCB : implements IXMLSelect, public CInterface
                 excptEntry->getStringX(len, message, "Message");
                 StringBuffer mstr(len, message);
                 rtlFree(message);
-                DBGLOG("Roxie exception: %s", mstr.str());
+                IERRLOG("Roxie exception: %s", mstr.str());
```

Comment:
Why IERRLOG, when is that used?  Is it used for logging an error for the programmer.  Should it be DERRLOG?

Reply:
I'm fine with IERRLOG if we all know that means internal/programmer.